pkg/k8s/client.go

```diff
@@ -45,22 +45,23 @@ import (
 
 var (
 	// ErrNilNode is returned when the Kubernetes API server has returned a nil node
-	ErrNilNode = goerrors.New("API server returned nil node")
-	client kubernetes.Interface
-	clusterName = "cilium-default"
+	ErrNilNode         = goerrors.New("API server returned nil node")
+	client             kubernetes.Interface
+	defaultClusterName = "k8sdefault"
```

Comment:
what about

`k8sdefault` -> `cilium-default`

and
`k8sClusterName = "k8s-default"`?

Reply:
Removed this in favor of the configuration value for the agent itself.